src/js/player.js

```diff
@@ -415,7 +416,12 @@ class Player extends Component {
     var techOptions = assign({
       'source': source,
       'playerId': this.id(),
-      'textTracks': this.textTracks_
+      'techId': `${this.id()}_${techName}_api`,
+      'textTracks': this.textTracks_,
+      'autoplay': this.options_.autoplay,
+      'preload': this.options_.preload,
+      'loop': this.options_.loop,
+      'muted': this.options_.muted
```

Comment:
Very nice. For #2033 I want to try to move away from using options like this and instead use the player properties (e.g. `this.autoplay()`), but there's still more work to do to make that even possible. Not something for this PR but just wanted to call it out as something that may change in the  near future.